package.json

```diff
@@ -166,7 +166,7 @@
     "tinyliquid": "^0.2.33",
     "url-loader": "^0.5.7",
     "uswds": "^1.0.0",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema#7ef76dfb7e334a23db5b663a8c0dff15747748f6",
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema#32580687bb7308e638a1d6ef675ca7a92cc4cd35",
```

Comment:
I think this has to do with why the 5490 tests are breaking. Check out [the diff](https://github.com/department-of-veterans-affairs/vets-json-schema/compare/7ef76dfb7e334a23db5b663a8c0dff15747748f6...32580687bb7308e638a1d6ef675ca7a92cc4cd35). The broken tests are all in `benefitHistory` which is what's modified between these two commits. (That's probably the problem with the schema test as well...)

Not that I know the _actual_ cause, mind you, but I'd look into the diff between the two commits in vets-json-schema.